classes/tables/entries_table.php

```diff
@@ -51,12 +64,18 @@ public function col_name($record) {
                 '/mod/cpdlogbook/delete.php',
                 ['cmid' => $this->cmid, 'id' => $record->id, 'sesskey' => sesskey()]
         );
+
+        $updatestr = get_string('update', 'mod_cpdlogbook');
```

Comment:
for reall common strings its better to reuse the existing ones in /lang/en/moodle which you can use without a plugin second arg eg 

get_string('update');